File: HeeeNoScreenShotView-demo/HeeeNoScreenShotView-demo-OC/HeeeNoScreenShotView-demo-OC/ViewController.m
//
//  ViewController.m
//  HeeeNoScreenShotView-demo-OC
//
//  Created by Heee on 2021/5/18.
//

#import "ViewController.h"
#import "HeeeNoScreenShotView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  
  if (@available(iOS 13.0, *)) {
    HeeeNoScreenShotView *noScreenShotView = [[HeeeNoScreenShotView alloc] initWithFrame:CGRectMake(50, 50, 240, 200)];
    noScreenShotView.backgroundColor = [UIColor colorWithWhite:0.95 alpha:1.0];
    [self.view addSubview:noScreenShotView];
    
    UIButton *privacyBtn = [[UIButton alloc] initWithFrame:CGRectMake(10, 30, 100, 40)];
    privacyBtn.backgroundColor = [UIColor systemRedColor];
    [privacyBtn setTitleColor:[UIColor whiteColor] forState:(UIControlStateNormal)];
    privacyBtn.titleLabel.font = [UIFont systemFontOfSize:15];
    [privacyBtn setTitle:@"隐私按钮" forState:(UIControlStateNormal)];
    [privacyBtn addTarget:self action:@selector(click) forControlEvents:(UIControlEventTouchUpInside)];
    [noScreenShotView addSubview:privacyBtn];
    
    UILabel *privacyLabel = [[UILabel alloc] initWithFrame:CGRectMake(130, 30, 100, 40)];
    privacyLabel.textAlignment = NSTextAlignmentCenter;
    privacyLabel.backgroundColor = [UIColor systemGreenColor];
    privacyLabel.font = [UIFont systemFontOfSize:15];
    privacyLabel.text = @"隐私文字";
    [noScreenShotView addSubview:privacyLabel];
  }
}

- (void)click {
  NSLog(@"点击了按钮");
}

@end

File: HeeeNoScreenShotView-demo/HeeeNoScreenShotView-demo-swift/HeeeNoScreenShotView-demo-swift/ViewController.swift
//
//  ViewController.swift
//  HeeeNoScreenShotView-demo-swift
//
//  Created by Heee on 2021/5/18.
//

import UIKit

class ViewController: UIViewController {

  override func viewDidLoad() {
    super.viewDidLoad()
    
    let noScreenShotView = HeeeNoScreenShotView.init(frame: .init(x: 50, y: 50, width: 240, height: 200))
    noScreenShotView.backgroundColor = .init(white: 0.95, alpha: 1.0)
    self.view.addSubview(noScreenShotView)
    
    let privacyBtn = UIButton.init(frame: .init(x: 10, y: 30, width: 100, height: 40))
    privacyBtn.backgroundColor = .systemRed
    privacyBtn.setTitle("隐私按钮", for: .normal)
    privacyBtn.setTitleColor(.white, for: .normal)
    privacyBtn.titleLabel?.font = .systemFont(ofSize: 15)
    privacyBtn.addTarget(self, action: #selector(click), for: .touchUpInside)
    noScreenShotView.addSubview(privacyBtn)
    
    let privacyLabel = UILabel.init(frame: .init(x: 130, y: 30, width: 100, height: 40))
    privacyLabel.textAlignment = .center
    privacyLabel.backgroundColor = .systemGreen
    privacyLabel.textColor = .black
    privacyLabel.font = .systemFont(ofSize: 15)
    privacyLabel.text = "隐私文字"
    noScreenShotView.addSubview(privacyLabel)
  }
  
  @objc func click() {
    print("点击了按钮")
  }
  
}

